domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -307,40 +307,32 @@ class TopicListControllerTest {
   }
 
   @Test
-  fun testRetrieveOngoingStoryList_defaultLesson_hasCorrectInfo() {
-    topicListController.getOngoingStoryList(profileId0).toLiveData()
-      .observeForever(mockOngoingStoryListObserver)
-    testCoroutineDispatchers.runCurrent()
-
-    verifyGetOngoingStoryListSucceeded()
-    verifyDefaultOngoingStoryListSucceeded()
-  }
-
-  @Test
-  @Ignore("Failing on Circle CI.")
-  fun testRetrieveOngoingStoryList_markRecentlyPlayedFracStory0Exp0_ongoingStoryListIsCorrect() {
-    storyProgressController.recordRecentlyPlayedChapter(
+  fun testRetrieveRecommendedActivityList_markChapterCompletedFracStory0Exp0_recommendedStoryListIsCorrect() { // ktlint-disable max-line-length
```

Comment:
@BenHenning Any suggestions how we can make this and other function names smaller to remove ktlint-disable check ?

Reply:
How about:

```kotlin
  fun testRetrieveRecommendedActivityList_markChapDoneFracStory0Exp0_storyListIsCorrect() {
```

This is at 91 characters so we have a bit of flexibility for the other test names if a bit more context needs to be added.

Reply:
Sounds good.